web_widget_bokeh_chart/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 ForgeFlow S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Web Widget Bokeh Chart",
+    "category": "Hidden",
+    "summary": "This widget allows to display charts using Bokeh library.",
+    "author": "ForgeFlow, " "Odoo Community Association (OCA), " "Creu Blanca",
+    "version": "18.0.1.0.0",
+    "maintainers": ["LoisRForgeFlow", "ChrisOForgeFlow"],
+    "development_status": "Production/Stable",
+    "website": "https://github.com/OCA/web",
+    "depends": ["web"],
+    "data": [],
+    "external_dependencies": {"python": ["bokeh==3.4.1"]},
```

Comment:
@JasminSForgeFlow could you take the occasion to upgrade library version to 3.6.3 (the last stable version https://pypi.org/project/bokeh/#history)?

Reply:
Shouldn't the names of the files related to the `src/lib/bokeh` library also be updated?

Reply:
Updated, Thanks